float64 t1
float64 t2
---
geometry_msgs/PoseWithCovariance pose_delta